tests/unit/fixtures/latedef-esnext.js

```diff
@@ -60,3 +60,9 @@ let dc = {
         return dc;
     }
 };
+
```

Comment:
Could you add a comment here referencing the regression at gh-2658 so future contributors understand the purpose of these statements?